roles/tomcat/tasks/main.yml

```
@@ -39,7 +39,7 @@
 - name: Download Tomcat
   get_url:
     url: "{{ dependencies_url.tomcat }}"
-    checksum: sha512:{{ dependencies_url.tomcat_sha512_checksum }}
+    checksum: sha512:{{ lookup('url', dependencies_url.tomcat_sha512_checksum_url).split(' ')[0] }}
```

Just one left

---

```
TASK [tomcat : Download Tomcat] ************************************************
fatal: [repository]: FAILED! => {"changed": false, "msg": "Unable to find a checksum for file 'apache-tomcat-9.0.54.tar.gz' in 'https://archive.apache.org/dist/tomcat/tomcat-9/v9.0.54/bin/apache-tomcat-9.0.54.tar.gz.sha512'"}
```

---

yes but we want to merge that after 1.2.0 is released and ansible min requirement is changed to 2.12 (2.10 & 2.11 doesn't look like sensible choices given it's mostly the same deprecation dates).
With that version we can merge with the lookup()/split(). Make sense?

---

Sorry to insist but with 2.12 the below should work:

```suggestion
    checksum: sha512:{{ dependencies_url.tomcat_sha512_checksum_url }}
```

---

ok, I schedule another rebase after merging #262 :D